### PR TITLE
BigQuery Plugin testcases

### PR DESCRIPTION
**Pull request by sawantpritam**
@tivv @rmstar @itsankit-google Please review.

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * constants.
+ */
+public class GCConstants {
+  public static final String ERROR_MSG_COLOR = "errorMessageColor";
+  public static final String ERROR_MSG_DATASET = "errorMessageDataset";
+  public static final String ERROR_MSG_TEMP_BUCKET = "errorMessageTemporaryBucket";
+  public static final String ERROR_MSG_REF = "errorMessageReference";
+  public static final String CDF_ATHENA_BQMTABLE = "cdf-athena-bq-bq-multitable";
+  public static final String A = "a";
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java]: why do you have this constant?

File: src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package io.cdap.plugin.utils;
+
+/**
+ * constants.
+ */
+public class GCConstants {
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java]: What is GCConstants? rename to something more meaningful?

File: src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+public class CdapUtils {
+
+  private static Properties pluginProperties = new Properties();
+  private static Properties errorProperties = new Properties();
+  private static final Logger logger = Logger.getLogger(CdapUtils.class);
+
+  static {
+
+    try {
+      pluginProperties.load(new FileInputStream("src/e2e-test/resources/pluginParameters.properties"));
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java]: load from classloader

File: src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import java.io.IOException;
+import java.util.Properties;
+
+import static io.cdap.plugin.utils.GCConstants.ERROR_MSG_COLOR;
+import static io.cdap.plugin.utils.GCConstants.ERROR_MSG_MANDATORY;
+
+/**
+ * CdapUtils contains the helper functions.
+ */
+public class CdapUtils {
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java]: use more descriptive name

File: src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java [hunk 5]
```diff
@@ -24,7 +24,7 @@
  */
 @RunWith(Cucumber.class)
 @CucumberOptions(
-        features = {"src/e2e-test/features"},
+        features = {"src/e2e-test/features/DemoGcs.feature"},
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java:27]: why do we have GCS test under plugin/gcp?

File: src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/package-info.java [hunk 6]
```diff
@@ -0,0 +1,4 @@
+/**
+ * comments.
```
Comment (rmstar) [inline src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/package-info.java]: please add meaningful comments

File: src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java (hunk 1 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java]: removed unused constants.

File: src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java (hunk 2 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java]: Renamed.

File: src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java (hunk 3 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java]: Changed to load from ClassLoader.

File: src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java (hunk 4 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/utils/CdapUtils.java]: Renamed.

File: src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/package-info.java (hunk 6 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/package-info.java]: Added comments

File: src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java (hunk 5 above)
Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java:27]: This folder was created only for Demo.

Comment (rmstar) [reply src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java:27]: The demo is done now, so let's move code to the right place. 
Tests for a GCS should be under src/e2e-test/java/io/cdap/plugin/gcs

Comment (sawantpritam) [reply src/e2e-test/java/io/cdap/plugin/gcp/tests/runner/TestRunner.java:27]: Sure. We will update this in GCS testcases PR.

**Comment by sawantpritam** [comment]
BigQuery e2e test scenarios added in  https://github.com/data-integrations/google-cloud/pull/886

**Pull request closed without merging**